### DISCUSSION TITLE
Contrato do validador e parâmetros de compilação

### DISCUSSION THREAD

**Discussion by ghost** [bacen/pilotord-kit-onboarding#50] [General]
Gostaria de saber o contrato do validador usado no bytecode do arquivo genesis e os parâmetros de compilação usados para checagem de confiabilidade

**Comment by aldenio** [comment]
Prezado @sandrogomes94 ,

Estamos utilizando os contratos do repositório da [Consensys](https://github.com/Consensys/permissioning-smart-contracts), conforme documentação do [besu](https://besu.hyperledger.org/stable/private-networks/tutorials/permissioning/onchain).


obs: caso sua dúvida tenha sido respondida, favor encerrar a issue.
